client/src/modules/exchange/exchange.js

```diff
@@ -0,0 +1,8 @@
+angular.module('bhima.controllers')
```

Comment:
This currently doesn't do anything.  Could we just remove it and have a route without a controller?  I think `ui-router` supports that.